.github/workflows/email-requester-onboarding-close.yml

```diff
@@ -0,0 +1,47 @@
+name: Send Message on Onboarding Issue Close
+
+on:
+  issues:
+    types:
+      - closed
+
+jobs:
+  send-message:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Check out repository
+        uses: actions/checkout@v2
```

Comment:
could this be updated to v4? 
actions/checkout@b4ffde65f46336ab88eb53be808477a3936bae11 # v4.1.1

Reply:
Amended